### ISSUE TITLE
Nullability annotations on type-parameter-typed declarations

### ISSUE THREAD

**Issue by yborovikov** [stylismo/nullability-annotations-inspection#11]
In current (`0.0.9`) version of the plugin the inspection complains about missing nullability annotations on fields and method parameters / return values whose type is a type parameter, e.g.:
```
public class Holder<T> {
  ...
  public T get() { ... }
}
```
While in Java 7 and older that could make sense, in Java 8 type parameters themselves can be annotated, so it could be a conscious choice of the class author to let the customer choose the nullability contract for each particular use case, e.g.:
```
Holder<@Nullable String> optionalStringHolder = ...
Holder<@NotNull String> requiredStringHolder = ...
```
It would be great if the plugin accounted for the open, non-final nature of type-parameter-typed declarations and allowed these without complaining. The behavior could be controlled by separate options for `Java 8+` and `Java 7-`.

Similarly, it could be useful to have an option to complain about missing type-use annotations in Java 8+ code, e.g. this code snippet could trigger a warning:
```
private final Holder<String> stringHolder = ... // missing type-use nullability annotation
```

Last but not least - thanks for this great inspection!

**Comment by stylismo** [comment]
Released as 0.0.10 and upoaded to JetBrains plugin repo: https://plugins.jetbrains.com/plugin/9418-nullability-annotations-inspection/update/41508

**Comment by stylismo** [comment]
Thanks Yegor, it's a really nice addition!
